src/pages/machine-learning/random-forest/index.md

```diff
@@ -2,14 +2,17 @@
 title: Random Forest
 ---
 ## Random Forest
+A Random Forest is a group of decision trees that make better decisions as a whole than individually.
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/machine-learning/random-forest/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+### Problem
+Decision trees by themselves are prone to **overfitting**. This means that the tree becomes so used to the training data that it has difficulty making decisions for data it has never seen before.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+### Solution with Random Forests
+Random Forests belong in the category of **ensemble learning** algorithms. This class of algorithms use many estimators to yield better results. This makes Random Forests usually **more accurate** than plain decision trees. In Random Forests, a bunch of decision trees are created. Each tree is **trained on a subset of the data and a subset of the features of that data**. This way the possibility of the estimators getting used to the data (overfitting) is greatly reduced, because **each of them work on the different data and features** than the others. This method of creating a bunch of estimators and training them on subsets of data is a technique in *ensemble learning* called **bagging** or *Bootstrap AGGregatING*. To get the prediction, the each of the decision trees vote on the correct prediction (classification) or they get the mean of their results (regression).
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+#### Extremely Randomized Trees (ExtraTrees)
```

Comment:
This part on extremely randomized trees might be good for a separate short article instead of having it here. Each article is supposed to be on one topic.

Reply:
Alright I'll remove it. Thanks!